python/samples/demos/guided_conversations/guided_conversation/utils/base_model_llm.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+import ast
+from types import NoneType
+from typing import get_args
+
+from pydantic import BaseModel, ValidationInfo, field_validator
+
+
+class BaseModelLLM(BaseModel):
```

Comment:
Can this be `KernelBaseModel` instead of `BaseModel`?

Reply:
Similar to Eduard's comment, I will leave this. The logic related to the artifacts can also be used outside of SK.